src/App.test.js

```diff
@@ -1,31 +1,15 @@
 import React from 'react';
-import { act } from 'react-dom/test-utils';
 import RepoPage from './content/RepoPage';
-import { mount } from 'enzyme';
-
-jest.mock('@octokit/core', () => {
-  return {
-    __esModule: true,
-    Octokit: jest.fn().mockImplementation(() => {
-      return {
-        request: () => {
-          return Promise.resolve({
-            status: 200,
-            data: [],
-          });
-        },
-      };
-    }),
-  };
-});
-
-it('renders a table with data and pagination', async () => {
-  const wrapper = mount(<RepoPage />);
-
-  expect(wrapper.find('.cds--pagination').length).toBe(0);
-
-  await act(() => new Promise(resolve => setTimeout(resolve, 0)));
-
-  wrapper.update();
-  expect(wrapper.find('.cds--pagination').length).toBe(1);
+import { render, screen, waitFor } from '@testing-library/react';
+
+it('renders a table with pagination', async () => {
+  const { container } = render(<RepoPage />);
+  expect(container.getElementsByClassName('.cds--pagination').length).toBe(0);
+  render(<RepoPage />);
+  await waitFor(
+    () => expect(screen.getByText('Items per page')).toBeInTheDocument(),
+    {
+      timeout: 1000,
+    }
+  );
```

Comment:
I would expect this to pass when testing it from the step 4 branch but here's the error I get when doing that:
<img width="1204" alt="image" src="https://github.com/carbon-design-system/carbon-tutorial/assets/40550942/fa458500-09c7-441c-a5df-8ade37803011">

Reply:
@francinelucca just increased the timeout to 2000 and they passed for me, let me know if you see something different. 🙏

Reply:
works now!